src/itkVkCommon.cxx

```diff
@@ -63,11 +72,16 @@ VkCommon::run(VkGPU * vkGPU, const VkParameters * vkParameters)
           vkGPU->device = deviceList[i];
           vkGPU->context = clCreateContext(NULL, 1, &vkGPU->device, NULL, NULL, &resCL);
           if (resCL != CL_SUCCESS)
+          {
+            std::cerr << __FILE__ "(" << __LINE__ << "): clCreateContext returned " << resCL << std::endl;
             return VKFFT_ERROR_FAILED_TO_CREATE_CONTEXT;
-          cl_command_queue commandQueue{ clCreateCommandQueue(vkGPU->context, vkGPU->device, 0, &resCL) };
+          }
+          vkGPU->commandQueue = clCreateCommandQueue(vkGPU->context, vkGPU->device, 0, &resCL);
           if (resCL != CL_SUCCESS)
+          {
+            std::cerr << __FILE__ "(" << __LINE__ << "): clCreateCommandQueue returned " << resCL << std::endl;
             return VKFFT_ERROR_FAILED_TO_CREATE_COMMAND_QUEUE;
-          vkGPU->commandQueue = commandQueue;
```

Comment:
Was this removed accidentally or on purpose?

Reply:
That is one of two statements that were merged into one.